assets/sass/05-blocks/utilities/_editor.scss

```diff
@@ -71,39 +69,19 @@
 }
 
 // Gutenberg Font-size utility classes
-.is-tiny-text,
-.has-tiny-font-size {
-	font-size: var(--global--font-size-xs);
-}
-
-.is-small-text,
-.has-small-font-size {
-	font-size: var(--global--font-size-sm);
-}
-
-.is-regular-text,
-.has-regular-font-size,
-.has-normal-font-size,
-.has-medium-font-size {
-	font-size: var(--global--font-size-md);
-}
-
 .is-large-text,
 .has-large-font-size {
-	font-size: var(--global--font-size-lg);
 	line-height: var(--global--line-height-heading);
```

Comment:
Please move the line heights with the font-sizes :). I meant that all references to the editor font size, the named font sizes such as large and gigantic, should not be inside `_editor.scss` but inside the new `font_sizes.scss` file.